### ISSUE TITLE
Reproduction Issues

### ISSUE THREAD

**Issue by itsnamgyu** [michuanhaohao/reid-strong-baseline#202]
I ran the code using the baseline.yml and found that the results were slightly below that reported on the README and original paper. Is this within the margin of error due to randomization, or am I missing something?

Would really appreciate if you could help me out @michuanhaohao @L1aoXingyu 

### Output

Ran on a single 2080ti

```sh
2021-07-11 14:18:21,932 reid_baseline.train INFO: Epoch[120] Iteration[180/183] Loss: 0.030, Acc: 1.000, Base Lr:
 3.50e-06
2021-07-11 14:18:23,825 reid_baseline.train INFO: Epoch 120 done. Time per batch: 30.902[s] Speed: 379.0[samples/
s]
2021-07-11 14:18:23,825 reid_baseline.train INFO: ----------
The test feature is normalized
2021-07-11 14:21:04,250 reid_baseline.train INFO: Validation Results - Epoch: 120
2021-07-11 14:21:04,250 reid_baseline.train INFO: mAP: 73.5%
2021-07-11 14:21:04,250 reid_baseline.train INFO: CMC curve, Rank-1  :87.3%
2021-07-11 14:21:04,250 reid_baseline.train INFO: CMC curve, Rank-5  :95.1%
2021-07-11 14:21:04,250 reid_baseline.train INFO: CMC curve, Rank-10 :96.7%
```

### Results

- **mAP**: 73.5 (vs 74.0 reported)
- **R1**: 87.3% (vs 87.7% reported)

### Details

- torch==1.9.0
- torchvision==0.10.0
- Used pre-trained model obtained from `resnet50(pretrained=True)`

**Comment by michuanhaohao** [comment]
Because the dataset is small-scale, the accuracy may fluctuate by no more than 1%. Yon can try different random seeds.

**Comment by itsnamgyu** [comment]
Thanks for the quick response. I'll try a few more runs!
